README.md

```diff
@@ -9,575 +7,336 @@
 
-# Deploy to minikube w/o registry.
-ko apply -L -f config/
+## Multi-Platform Images
 
-# This is the same as above.
-KO_DOCKER_REPO=ko.local ko apply -f config/
-```
+Because Go supports cross-compilation to other CPU architectures and operating systems, `ko` excels at producing multi-platform images.
 
-A caveat of this approach is that it will not work if your container is
-configured with `imagePullPolicy: Always` because despite having the image
-locally, a pull is performed to ensure we have the latest version, it still
-exists, and that access hasn't been revoked. A workaround for this is to use
-`imagePullPolicy: IfNotPresent`, which should work well with `ko` in all
-contexts.
+To build and push an image for all platforms supported by the configured base image, simply add `--platform=all`. This will instruct `ko` to look up all the supported platforms in the base image, execute `GOOS=<os> GOARCH=<arch> GOARM=<variant> go build` for each platform, and produce a manifest list containing an image for each platform.
 
-Images will appear in the Docker daemon as
-`ko.local/import.path.com/foo/cmd/bar`. With `--local` import paths are always
-preserved (see `--preserve-import-paths`).
+You can also select specific platforms, for example, `--platform=linux/amd64,linux/arm64`
 
-## With `kind`
+## Static Assets
 
-Likewise, you can use `ko` with [kind](https://github.com/kubernetes-sigs/kind)
-to aid in rapid local iteration both locally and in small CI environments. To
-instruct `ko` to publish images into your `kind` cluster, the `KO_DOCKER_REPO`
-variable must be set to `kind.local`.
+`ko` can also bundle static assets into the images it produces.
 
-This would look something like:
+By convention, any contents of a directory named `<importpath>/kodata/` will be bundled into the image, and the path where it's available in the image will be identified by the environment variable `KO_DATA_PATH`.
 
-```shell
-# Create a kind cluster
-kind create cluster
+As an example, you can bundle and serve static contents in your image:
 
-# Deploy to kind w/o registry.
-KO_DOCKER_REPO=kind.local ko apply -f config/
+```
+cmd/
+  app/
+    main.go
+    kodata/
+      favicon.ico
+      index.html
 ```
 
-If you want to create a `kind` cluster with a non default name, you can set the
-`KIND_CLUSTER_NAME` variable to the respective name (which is also supported by
-[`kind` itself](https://github.com/kubernetes-sigs/kind/releases/tag/v0.8.0)).
-
-Like with `minikube` above, a caveat of this approach is that it will not work
-if your container is configured with `imagePullPolicy: Always` because despite
-having the image locally, a pull is performed to ensure we have the latest
-version, it still exists, and that access hasn't been revoked. A workaround for
-this is to use `imagePullPolicy: IfNotPresent`, which should work well with `ko`
-in all contexts.
+Then, in your `main.go`:
+```go
+func main() {
```

Comment:
I wish this example had been there two weeks ago!

Reply:
yeah, it is a pretty good pattern: https://github.com/knative-party/site/blob/main/site.go